src/player.cpp

```diff
@@ -12119,51 +12119,26 @@ int player::encumb( body_part bp ) const
  */
 int player::encumb(body_part bp, double &layers, int &armorenc) const
 {
-    int ret = 0;
-    double layer[MAX_CLOTHING_LAYER] = { };
-    int level = 0;
-    bool is_wearing_active_power_armor = false;
-    for( auto &w : worn ) {
-        if( w.active && w.is_power_armor() ) {
-            is_wearing_active_power_armor = true;
-            break;
-        }
-    }
+    int layer[MAX_CLOTHING_LAYER] = { };
 
-    for( auto& w : worn ) {
-        if( !w.covers(bp) ) {
-            continue;
-        }
+    for (auto w = worn.begin(); w != worn.end(); ++w) {
```

Comment:
i had implemented further functions to test for layer misplacement which would need either an index or iter to find the specific item in the players inventory but this is perhaps redundant for now considering #12741